pkgs/tools/networking/rosenpass/tools.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, makeWrapper
+, stdenv
+, coreutils
+, findutils
+, gawk
+, rosenpass
+, wireguard-tools
+}:
+stdenv.mkDerivation {
+  pname = "rosenpass-tools";
+  inherit (rosenpass) version src;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postInstall =
+    let
+      rpDependencies = [
+        coreutils
+        findutils
+        gawk
+        rosenpass
+        wireguard-tools
+      ];
+    in
+    ''
+      install -D $src/rp $out/bin/rp
+      install -D $src/doc/rp.1 $out/share/man/man1/rp.1
+      wrapProgram $out/bin/rp --prefix PATH : ${lib.makeBinPath rpDependencies}
```

Comment:
```suggestion
      wrapProgram $out/bin/rp \
        --prefix PATH : ${lib.makeBinPath [ coreutils findutils gawk rosenpass wireguard-tools ];}
```

Reply:
`envsubst.meta.mainProgram` is not set, I'll set it in this PR.